components/resourceDetails/__snapshots__/index.test.tsx.snap

```diff
@@ -869,7 +869,7 @@ font-display: block;",
             className="PodBrowser-content-h3"
             title="/container/"
           >
-            Name
+            container
```

Comment:
Let's change the test (or create another one) that has space in the name (or some other characters we want to decode).